src/reason_parser.mly

```diff
@@ -788,8 +788,17 @@ let rec string_of_longident = function
 
 let built_in_explicit_arity_constructors = ["Some"; "Assert_failure"; "Match_failure"]
 
+let jsx_component module_name attrs children ~loc =
```

Comment:
Wondering if we should call it `RSX` since we've already deviated a little bit from `JSX` (like allowing any simple expression for attributes etc).